src/arch/xtensa/configs/imx8_defconfig

```diff
@@ -1,3 +1,4 @@
 CONFIG_IMX8=y
 CONFIG_HAVE_AGENT=n
 CONFIG_FORMAT_CONVERT_HIFI3=n
+CONFIG_KPB_FORCE_COPY_TYPE=n
```

Comment:
I feel like we can make a depends on !DUMMY_DMA instead of changing the kconfig fragment... I can accept this as well.

Reply:
@paulstelian97 I would prefer to go with this for now. We can change it in the future if that is too much of a burden.